R/udunits.R

```diff
@@ -1,3 +1,16 @@
+#' Test if two units are convertible
+#'
```

Comment:
Please, add a description, stating that the function directly interfaces UDUNITS-2.

Reply:
Sorry, I wasn't clear enough. I meant a description in a separate line. In roxygen2, the first line is the title of the help page, and the next paragraph is taken as the description. See, e.g., https://github.com/r-quantities/units/blob/master/R/user_conversion.R#L1-L5

Reply:
Don't mind, didn't see the line above the comment.